### ISSUE TITLE
[Question] New permissions on the GitHub App

### ISSUE THREAD

**Issue by sjparkinson** [renovatebot/config-help#358]
### Which Renovate are you using? CLI, App, or Pro

The GitHub App.

### Which platform are you using? GitHub, GitLab, Bitbucket Azure DevOps

GitHub

### Have you checked the logs? Don't forget to include them if relevant

![image](https://user-images.githubusercontent.com/51677/64943152-7ef2ac00-d85a-11e9-8bd5-6e3c8bbfc5f1.png)

### What is your question?

I'd like to understand why these new permissions are being requested before we accept them.

**Comment by rarkins** [comment]
The new permissions are kind of independent of each other. Descriptions follow:

Read-only access to members: Renovate's dashboard (https://app.renovatebot.com/dashboard) will soon support the ability for administrators of an org to enable org-wide settings such as default onboarding configs as well as encrypted secrets like CI systems have. But for us to know that someone is an admin of the org, we need this "members" permission. Without it, nobody will be recognised as an admin and hence nobody can set org-wide options. I've contacted GitHub in case there is some other way to know if a logged-in user is an admin or not, and will remove this permission if indeed there is a way.

Email addresses: Until now we haven't asked for access to user email addresses but it's turned out to be a problem because we have no way of contacting people if we need to, such as to notify of faulty configs or if we were to change policies. There have been instance of accounts we've needed to block because of a problem and we're not able to reach the user. Posting an issue into one of their public repos to get attention would be really bad form too. We haven't written the code for this but we will gather email addresses only once people log into the dashboard and we will display a consent form at that time, so we're compliant with everything we need to be. 

In short, we need to know who's an admin of an org in order to enable admin-only privileges, and we need to be able to contact users.

**Comment by rarkins** [comment]
FYI here's the view from the app settings:

![image](https://user-images.githubusercontent.com/6311784/64962781-52f61b80-d898-11e9-893c-e66d5da91b12.png)

![image](https://user-images.githubusercontent.com/6311784/64962800-5f7a7400-d898-11e9-84e4-b7c3793e15de.png)

**Comment by sjparkinson** [comment]
Beautiful, thanks very much Rhys. Sounds like great changes too in the dashboard!
